README.md

```diff
@@ -172,6 +172,7 @@ additional details on each available environment variable.
 | `ECS_ENABLE_TASK_CPU_MEM_LIMIT` | `true` | Whether to enable task-level cpu and memory limits | `true` | `false` |
 | `ECS_CGROUP_PATH` | `/sys/fs/cgroup` | The root cgroup path that is expected by the ECS agent. This is the path that accessible from the agent mount. | `/sys/fs/cgroup` | Not applicable |
 | `ECS_CGROUP_CPU_PERIOD` | `10ms` | CGroups CPU period for task level limits. This value should be between 8ms to 100ms | `100ms` | Not applicable |
+| `AGENT_HEALTHCHECK_LOCALHOST_OVERRIDE` | `127.0.0.1` | Override for the ecs-agent container's healthcheck localhost ip address| `localhost` | `localhost` |
```

Comment:
This env var should be prepended with `ECS_`

and nit that I would prefer it less verbose, maybe just `ECS_AGENT_HEALTHCHECK_HOST`? with default value `localhost`